otcextensions/sdk/dms/v1/group.py

```diff
@@ -56,17 +55,17 @@ class Group(_base.Resource):
     #: *Type: int*
     available_deadletters = resource.Body('available_deadletters', type=int)
 
-    def create(self, session, group):
+    def create(self, session, *args, **kwargs):
         """create group"""
-        body = {"groups": [{'name': group}]}
+        body = {"groups": [{'name': self.name}]}
```

Comment:
do we support only single group creation?
AFAIK this API supports multiple group creation
At least they state it in documentation:
_Multiple consumer groups can be created for a queue at a time._

Reply:
Yes on both: API supports, we do not, and we are not going to change this now (no real usage)